socorro/external/crashstorage_base.py

```diff
@@ -589,13 +590,27 @@ def save_processed(self, processed_crash):
     #--------------------------------------------------------------------------
     def save_raw_and_processed(self, raw_crash, dump, processed_crash,
                                crash_id):
+        storage_exception = PolyStorageError()
         for a_store in self.stores.itervalues():
-            a_store.save_raw_and_processed(
-              raw_crash,
-              dump,
-              processed_crash,
-              crash_id
-            )
+            self.logger.debug('writing to %s' % (
+                hasattr(a_store, 'wrapped_object') and
```

Comment:
This log has been very helpful while debugging yesterday, but yeah, maybe we can do with less noise and just add it whenever we need it. I'll actually keep it in there, but commented out, so that we can easily enable it live.